.circleci/config.yml

```diff
@@ -598,6 +600,8 @@ jobs:
           root: ~/fun
           paths:
             - src/frontend/i18n/frontend.pot
+            - src/richie/static/richie/js/*.js
```

Comment:
The build is not made in production mode, you will not have an optimized build. Is it impostant or not ?

Reply:
Yes it is. And it should as it will be the default build that will be shipped with the python package.

Reply:
This task name is not relevant. It tests that the front application build well and we need the build in development mode.

You can create a new task or add more steps to this task to build the app in production mode.

Reply:
True. Done.